test/functional/apps/management/data_views/_scripted_fields.ts

```diff
@@ -155,7 +155,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         it('should see scripted field value in Discover', async function () {
           await PageObjects.common.navigateToApp('discover');
 
-          await PageObjects.unifiedFieldList.clickFieldListItem(scriptedPainlessFieldName);
```

Comment:
I encountered some flakiness with one or two of these tests, and it seemed to be because we were opening the field popover before clicking the inline "add" button, which wasn't required, and the click was sometimes being intercepted by the popover arrow element instead of the button. Dropping these lines seems to fix the flakiness.